### ISSUE TITLE
call-backs from Dash-plotly to JupyterLab

### ISSUE THREAD

**Issue by Shai2u** [plotly/jupyter-dash#54]
![Screen Shot 2021-02-12 at 7 45 19](https://user-images.githubusercontent.com/19770769/107736067-d1632100-6d09-11eb-956c-9b05b73eb71f.png)
Hi,
I am trying to incorporate ArcGIS WebScenes in my Dash-plotly workflow.
I think I have two main options 

1. To use iFrame and I louse the call back (As far as I know), which looks great but I am very limited in modification and call back. 
2. To incorporate Dash-Plotly Call backs to Jupyer-Lab like in the example in the attached image, **I was able one time to have a call-back from the map to the output view on the left side**.  

Obviously I prefer the Dash-plotly--> Jupyter Lab callback option.
I have much more control in modifying the websene in the Jupyter-Lab embedded cell (moving camera, changing visualisation.  Even though I can modify the websene and update the iframe, it's performance is inferrier to that of an embed websene like in the jupyte-lab.

So in Short - I am looking to send a call back from Dash-plotly back to Jupyter Lab, and **I want it to work more than once**. (It's now a one time thing)  I tried both 'Inline' and the 'JupyterLab' option. 

Thanks in advance.
